patch-diff-report-tool/src/main/resources/maven-theme.css

```diff
@@ -122,6 +122,9 @@ table.bodyTable td.c {
   background-color: #eee;
 }
 
+.wrap-anywhere {
```

Comment:
Why have a class name and just wrap all TDs/THs?

Reply:
If we apply such class to some html tag, it should work. We do have long values in headers of table (full name of module)

Reply:
If you change `.wrap-anywhere {` in the CSS to just `td, th {` it will apply to all TDs/THs in the document the CSS file applies to. Then you can just remove hardcoding the class name in the HTML.

Reply:
it will result in https://romani.github.io/wrap-all-td/diff/guava/index.html
wrap of all, even short columns, that are better to not break
![Screenshot_2019-12-20 checkstyle xml difference report](https://user-images.githubusercontent.com/812984/71289192-5bd80c80-2321-11ea-81ca-26ed460d729f.png)

Reply:
Ok, nevermind.